system/Database/Forge.php

```diff
@@ -151,7 +151,7 @@ class Forge
 	 *
 	 * @var string
 	 */
-	protected $null = '';
+	protected $_null = '';
```

Comment:
I'm torn here if I should deprecate this, or rename since this is the actual prop name used by extensions.

Reply:
It's more work, but I think we should follow the naming convention from the "base" Forge class. Besides, from what I remember we're trying to eliminate underscores from the variable names, so it would be a step forward as well. 

This is an internal variable, so I don't think there will be any side effects.

We should simply treat this as a bug - no need for depreciation.

Reply:
Let's go ahead and mark this `@internal` if we're going to treat it that way.